docs/en/observability/logs-plaintext.asciidoc

```diff
@@ -0,0 +1,243 @@
+
+Install {filebeat} on the server you want to monitor by running the commands that align with your system:
+
+include::tab-widgets/logs/filebeat-install-widget.asciidoc[]
+
+[discrete]
+[[step-2-plaintext-connect-to-your-project]]
+=== Step 2: Connect to {es}
+
+Connect to {es} using an API key to set up {filebeat}. Set the following information in the `filebeat.yml` file:
+
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["your-projects-elasticsearch-endpoint"]
+  api_key: "id:api_key"
+----
+
+. Set the `hosts` to your deployment's {es} endpoint. Copy the {es} endpoint from your deployment's page and add the port (the default port is `443`). For example, `https://my-deployment.es.us-central1.gcp.cloud.es.io:443`.
++
+--
+[role="screenshot"]
+image::images/es-endpoint-cluster-id.png[{es} endpoint and cluster id location, 50%]
+--
+. From **Developer tools**, run the following command to create an API key that grants `manage` permissions for the `cluster` and the `filebeat-*` indices using:
++
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "filebeat_host001",
+  "role_descriptors": {
+    "filebeat_writer": {
+      "cluster": ["manage"],
+      "index": [
+        {
+          "names": ["filebeat-*"],
+          "privileges": ["manage", "create_doc"]
+        }
+      ]
+    }
+  }
+}
+----
++
+Refer to {filebeat-ref}/beats-api-keys.html[Grant access using API keys] for more information.
+
+[discrete]
+[[step-3-plaintext-configure-filebeat]]
+=== Step 3: Configure {filebeat}
+
+Add the following configuration to your `filebeat.yaml` file to start collecting log data.
+
+[source,yaml]
+----
+filebeat.inputs:
+- type: filestream  <1>
+  enabled: true
+  paths: /path/to/logs.log  <2>
+----
+<1> Reads lines from an active log file.
+<2> Paths that you want {filebeat} to crawl and fetch logs from.
+
+[discrete]
+[[step-4-plaintext-set-up-and-start-filebeat]]
+=== Step 4: Set up and start {filebeat}
+
+{filebeat} comes with predefined assets for parsing, indexing, and visualizing your data. To load these assets:
+
+From the {filebeat} installation directory, set the {ref}/index-templates.html[index template] by running the command that aligns with your system:
+
+include::tab-widgets/logs/filebeat-setup-widget.asciidoc[]
+
+From the {filebeat} installation directory, start filebeat by running the command that aligns with your system:
+
+include::{beats-root}/libbeat/docs/tab-widgets/start-widget-filebeat.asciidoc[]
+
+[discrete]
+[[step-5-plaintext-parse-logs-with-an-ingest-pipeline]]
+=== Step 5: Parse logs with an ingest pipeline
+
+use an ingest pipeline to parse the contents of your logs into structured, {ecs-ref}/ecs-reference.html[Elastic Common Schema (ECS)]-compatible fields.
+
+Create an ingest pipeline with a {ref}/dissect-processor.html[dissect processor] to extract structured ECS fields from your log messages. In your project, navigate to **Developer Tools** and using a command similar to the following example:
```

Comment:
Wording here makes it sound like the dissect processor creates the ingest pipeline. Maybe say "Create an ingest pipeline that defines a dissect processor" or "that includes a dissect processor."